app.js

```diff
@@ -29,7 +29,7 @@ winston.add(
 const models = require('./database/models');
 
 // Sync Database
-models.sequelize.sync().then(function() {
+models.sequelize.sync({force: true}).then(function() {
```

Comment:
if this is what you referred to that day, please remove it.  It should not be the the branch. You can keep using it for dev purposes.

Reply:
Okay! Corrected it.